source/transactions-convenient-api/tests/commit-retry.yml

```diff
@@ -270,3 +257,53 @@ tests:
       collection:
         data:
           - { _id: 1 }
+  -
+    description: commit is not retried after MaxTimeMSExpired error
```

Comment:
Does it make sense to configure a fail point for MaxTimeMSExpired without adding `maxTimeMS` to the outgoing commit command? I realize it probably doesn't matter since the driver's behavior when processing a response/error shouldn't be affected by the original outgoing command, but doing so might improve readability if someone comes across this down the line.

If this would be too much trouble (since you inherit `withTransaction` as one block), then perhaps a comment about that explaining that we're not bothering to specify `maxTimeMS` as a command option would work just as well for readability.

Reply:
Yeah that makes sense. I added maxTimeMS.